pages/register.tsx

```diff
@@ -45,11 +49,34 @@ export default function Register() {
     //setting user specific initial values
     formInitialValues['id'] = user?.id || '';
     formInitialValues['preferredEmail'] = user?.preferredEmail || '';
-    formInitialValues['firstName'] = user?.firstName || '';
+    formInitialValues['firstName'] = user?.firstName?.split(' ')[0] || '';
```

Comment:
What is this supposed to do? Should we discard first names that are multiple words?
If this instead trying to trim the string, it should use [`trimEnd`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd).

Reply:
Google treats the first name option as your full name on your google account. So if someone would sign in with google, it would autofill the first name field to your full name. There were some instances of people from hackutd IX not noticing it when registering so their profile names had their last name on twice. I just had it auto fill to the first name associated with the google account to make it easy. If there's a better idea on autofilling it lmk

Reply:
Okay, I'm not super familiar with how this value is being seeded, so I'll research and come back to this later.